.gitignore

```diff
@@ -52,6 +52,7 @@ library/library_example
 library/recommend
 library/gd_mf_weights
 library/test_search
+library/search_generate
```

Comment:
I'm not sure about this, but it would appear that search_generate is a new generated file.  @JohnLangford please correct me if I'm wrong.